src/WooCommerce.php

```diff
@@ -909,7 +909,7 @@ public static function woocommerce_validate_postcode($valid, $postcode, $country
         break;
 
       case 'LI':
-        $valid = (bool) preg_match('@^(948[5-9])|(949[0-7])$@', $postcode);
+        $valid = (bool) preg_match('@^((948[5-9]{1})|(949[0-7]{1}))$@', $postcode);
```

Comment:
`{1}` is the default of each character.
```suggestion
        $valid = (bool) preg_match('@^((948[5-9])|(949[0-7]))$@', $postcode);
```

Reply:
It is added to limit to 1 character length, it otherwise allows 5 digit postcodes, I have already tested this.

Reply:
I do not see a repeater instruction in the expression, so this does not match 5 digits.

See for yourself here: https://regex101.com/r/jIZPN7/1